Packs/CommonScripts/Scripts/GenerateAsBuiltConfiguration/GenerateAsBuiltConfiguration.yml

```diff
@@ -0,0 +1,34 @@
+args:
+- description: Who has prepared this as-built.
+  name: author
+- description: Whom this as-built was prepared for.
+  name: customer
+- defaultValue: '7'
+  description: Days back to search for incidents.
+  name: days
+- description: Maximum number of results to return - default 1000.
+  name: size
+- description: Retrieve more information such as dependencies from a specific use case. For multiple playbooks enter comma-separated values.
```

Comment:
A comma-separated list of.....

not sure what information user is supposed to put here.